src/sentry/seer/similarity/utils.py

```diff
@@ -459,12 +459,15 @@ def event_content_is_seer_eligible(event: GroupEvent | Event) -> bool:
     return True
 
 
-def killswitch_enabled(project_id: int, event: GroupEvent | Event | None = None) -> bool:
+def killswitch_enabled(
+    project_id: int | None,
+    event: GroupEvent | Event | None = None,
+) -> bool:
     """
     Check both the global and similarity-specific Seer killswitches.
     """
 
-    logger_extra = {"event_id": event.event_id if event else None, "project_id": project_id}
+    logger_extra = {"event_id": event.event_id if event else None, "project_id": project_id or ""}
```

Comment:
Super-tiny nit, def not worth changing unless you're making other changes:

```suggestion
    logger_extra = {"event_id": event.event_id if event else None, "project_id": project_id or None}
```

To match event id?